src/lib/logger.ts

```diff
@@ -21,17 +22,25 @@ const additionalProperties = {
 
 export const log = {
     exception: (error: Error | undefined) => {
-        if (error && (error.message.includes('Failed to fetch') || error.message.includes('Load failed'))) {
+        if (
+            error &&
+            (error.message.includes('Failed to fetch') ||
+                error.message.includes('Load failed') ||
+                error.message.includes('NetworkError when attempting to fetch'))
+        ) {
             // Don't log network errors to app insights (since they can't be avoided)
             console.error(error);
         } else if (!browserSupported) {
             // Don't log errors of unsupported browsers to app insights (since they can't be avoided)
             console.error(error);
         } else if (error) {
+            const { url, cacheBustVersion } =
+                'url' in error ? (error as WellFetchError) : { url: null, cacheBustVersion: null };
             appInsights.trackException(
                 { exception: error },
                 {
                     ...additionalProperties,
+                    ...(url ? { url, cacheBustVersion } : null),
```

Comment:
All the above stuff is to make this work, being able to include the url and cacheBustVersion when we hit an error.